src/DataStructures/Python/DataVector.cpp

```diff
@@ -49,6 +49,46 @@ void bind_datavector() {
       .def(bp::self += bp::other<double>{})
       // Need to do math explicitly converting to DataVector because we don't
       // want to represent all the possible expression template types
+      .def("abs", +[](const DataVector& t) { return DataVector{abs(t)}; })
+      .def("acos", +[](const DataVector& t) { return DataVector{acos(t)}; })
+      .def("acosh", +[](const DataVector& t) { return DataVector{acosh(t)}; })
+      .def("asin", +[](const DataVector& t) { return DataVector{asin(t)}; })
+      .def("asinh", +[](const DataVector& t) { return DataVector{asinh(t)}; })
+      .def("atan", +[](const DataVector& t) { return DataVector{atan(t)}; })
+      .def("atan2", +[](const DataVector& t,
```

Comment:
naming `t, v` `y, x` will make it easier to figure out which is which in the future. For single arg functions the name is not so important for these type of forwarding functions.